src/views/Course.jsx

```diff
@@ -0,0 +1,17 @@
+import CoursesHero from "../components/Courses/CoursesHero";
```

Comment:
You are using /Courses instead of /Course

Reply:
Fixed this